r-pkg/tests/testthat/test-repo_characteristics.R

```diff
@@ -35,7 +35,7 @@ test_that('R CMD check should not return any unexpected errors, warnings, or not
     testthat::skip_on_cran()
     
     # Check the package
-    x <- devtools::check(pkg = '../../../uptasticsearch'
+    x <- devtools::check(pkg = '../../../r-pkg'
```

Comment:
Okay. I figured it out. If you run devtools::test(), it passes with this change. However, if you run devtools::check(), it does not because the working directory is different. We could set the working directory before this line.